### PR TITLE
Bump dependencies for php 8.1 support

### PR DESCRIPTION
**Pull request by come-nc**

### CODE REVIEW

File: composer/autoload_classmap.php [hunk 1]
```diff
@@ -2066,6 +2066,15 @@
...
+    'Pimple\\Tests\\Fixtures\\Invokable' => $vendorDir . '/pimple/pimple/src/Pimple/Tests/Fixtures/Invokable.php',
+    'Pimple\\Tests\\Fixtures\\NonInvokable' => $vendorDir . '/pimple/pimple/src/Pimple/Tests/Fixtures/NonInvokable.php',
+    'Pimple\\Tests\\Fixtures\\PimpleServiceProvider' => $vendorDir . '/pimple/pimple/src/Pimple/Tests/Fixtures/PimpleServiceProvider.php',
+    'Pimple\\Tests\\Fixtures\\Service' => $vendorDir . '/pimple/pimple/src/Pimple/Tests/Fixtures/Service.php',
+    'Pimple\\Tests\\PimpleServiceProviderInterfaceTest' => $vendorDir . '/pimple/pimple/src/Pimple/Tests/PimpleServiceProviderInterfaceTest.php',
+    'Pimple\\Tests\\PimpleTest' => $vendorDir . '/pimple/pimple/src/Pimple/Tests/PimpleTest.php',
+    'Pimple\\Tests\\Psr11\\ContainerTest' => $vendorDir . '/pimple/pimple/src/Pimple/Tests/Psr11/ContainerTest.php',
+    'Pimple\\Tests\\Psr11\\ServiceLocatorTest' => $vendorDir . '/pimple/pimple/src/Pimple/Tests/Psr11/ServiceLocatorTest.php',
+    'Pimple\\Tests\\ServiceIteratorTest' => $vendorDir . '/pimple/pimple/src/Pimple/Tests/ServiceIteratorTest.php',
```
Comment (nickvergessen) [inline composer/autoload_classmap.php:2077]: add to gitignore

Comment (come-nc) [reply composer/autoload_classmap.php:2077]: There is already `pimple/pimple/src/Pimple/Tests` in the gitignore

Comment (nickvergessen) [reply composer/autoload_classmap.php:2077]: but you had the files locally before running the class mapper?

Comment (come-nc) [reply composer/autoload_classmap.php:2077]: I guess so, I don’t know, I just used:
```
composer require pimple/pimple:^3.5.0
composer install --no-dev
```
What is the missing step to ignore tests then?

Comment (come-nc) [reply composer/autoload_classmap.php:2077]: I tried to add:
```
git clean -X -d -f
composer dump-autoload
```
It does remove the tests from the autoload.
But still the CI complains: https://github.com/nextcloud/3rdparty/pull/905

**Comment by come-nc** [comment]
Done in several PR instead

**Pull request closed without merging**